articles/the-future-of-money-private-innovation-issues-paper-a-guide-for-kiwi-bitcoiners.html

```diff
@@ -0,0 +1,192 @@
+  <meta content="The Future of Money - Private Innovation Issues Paper - A Guide for Kiwi Bitcoiners" property="og:title">
+  <meta content="This is a guide for Kiwi Bitcoiners on the Reserve Bank of New Zealand&#x27;s request for feedback on their issues paper - The Future of Money - Private Innovation / Te Moni Anamata - Te Auahatanga. What it’s about, a condensed version of the paper, what you have to do and some handy resources." property="og:description">
+  <meta content="/images/ogp.jpg" property="og:image">
+  <meta content="The Future of Money - Private Innovation Issues Paper - A Guide for Kiwi Bitcoiners" property="twitter:title">
+  <meta content="This is a guide for Kiwi Bitcoiners on the Reserve Bank of New Zealand&#x27;s request for feedback on their issues paper - The Future of Money - Private Innovation / Te Moni Anamata - Te Auahatanga. What it’s about, a condensed version of the paper, what you have to do and some handy resources." property="twitter:description">
+  <meta content="/images/ogp.jpg" property="twitter:image">
+  <meta property="og:type" content="website">
+  <meta content="summary_large_image" name="twitter:card">
+  <meta content="width=device-width, initial-scale=1" name="viewport">
+  <link href="../css/normalize.css" rel="stylesheet" type="text/css">
+  <link href="../css/components.css" rel="stylesheet" type="text/css">
+  <link href="../css/nz-bitcoiners.css" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com" rel="preconnect">
+  <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin="anonymous">
+  <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js" type="text/javascript"></script>
+  <script type="text/javascript">
+      WebFont.load({
+        google: {
+          families: ["Open Sans:300,300italic,400,400italic,600,600italic,700,700italic,800,800italic"]
+        }
+      });
+    </script>
+  <!-- [if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js" type="text/javascript"></script><![endif] -->
+  <script type="text/javascript">
+      ! function(o, c) {
+        var n = c.documentElement,
+          t = " w-mod-";
+        n.className += t + "js", ("ontouchstart" in o || o.DocumentTouch && c instanceof DocumentTouch) && (n.className += t + "touch")
+      }(window, document);
+    </script>
+  <link href="../images/favicon.png" rel="shortcut icon" type="image/x-icon">
+  <link href="../images/webclip.png" rel="apple-touch-icon">
+</head>
+<body class="body">
+  <div class="nav-section wf-section">
+    <div data-animation="default" data-collapse="medium" data-duration="400" data-easing="ease" data-easing2="ease" role="banner" class="navbar w-nav">
+      <div class="container w-container">
+        <a href="http://../index.html" class="nav-logo w-nav-brand"><img src="../images/nzbitcoiners-logo.svg" loading="lazy" alt="NZ Bitcoiners logo."></a>
```

Comment:
This is broken currently. I assume the new version for the articles-guides.html would work if pasted here.

Reply:
Sends navigation to the literal address ../index.html